skimage/measure/tests/test_regionprops.py

```diff
@@ -198,51 +338,75 @@ def test_coordinates():
     prop_coords = regionprops(sample)[0].coords
     assert_array_equal(prop_coords, coords)
 
+    spacing = (1, 0.5)
+    prop_coords = regionprops(sample, spacing=spacing)[0].coords
+    assert_array_equal(prop_coords, coords * np.array(spacing))
```

Comment:
For all of these tests, I'd prefer if there was a separate test function for unit spacing, isotropic spacing, and anisotropic spacing, perhaps using `pytest.parametrize`? It makes it much easier to find failing tests in the future...

Reply:
Makes sense. 

You will also notice that not all features are tested with anisotropy, because I could not find reference numbers. Do you know of some implementations that could be used?

Reply:
I don't. Typically I would try to hand-check the numbers (going step by step in the implementation) and then save those as a reference. Or you can derive some from first principles, for example, drawing an ellipse with a particular orientation using `skimage.draw`, then taking away every other row of pixels and using `spacing=(2, 1)` and making sure that the results are close to the isotropic case.